joint_limits:
  arm_joint_1:
    has_position_limits: false
    # min_position: -2.967059728
    # max_position: 2.967059728
    has_velocity_limits: true
    max_velocity: 1.112
    has_acceleration_limits: true
    max_acceleration: 4.16
  arm_joint_2:
    has_position_limits: true
    min_position: -2.41
    max_position: 2.41
    has_velocity_limits: true
    max_velocity: 1.112
    has_acceleration_limits: true
    max_acceleration: 4.16
  arm_joint_3:
    has_position_limits: false
    # min_position: -2.094395102
    # max_position: 2.635447171
    has_velocity_limits: true
    max_velocity: 1.112
    has_acceleration_limits: true
    max_acceleration: 4.16
  arm_joint_4:
    has_position_limits: true
    min_position: -2.66
    max_position: 2.66
    has_velocity_limits: true
    max_velocity: 1.112
    has_acceleration_limits: true
    max_acceleration: 4.16
  arm_joint_5:
    has_position_limits: false
    # min_position: -2.094395102
    # max_position: 2.094395102
    has_velocity_limits: true
    max_velocity: 0.976
    has_acceleration_limits: true
    max_acceleration: 8.0
  arm_joint_6:
    has_position_limits: true
    min_position: -2.23
    max_position: 2.23
    has_velocity_limits: true
    max_velocity: 0.976
    has_acceleration_limits: true
    max_acceleration: 8.0
  arm_joint_7:
    has_position_limits: false
    # min_position: -6.283185
    # max_position: 6.283185
    has_velocity_limits: true
    max_velocity: 0.976
    has_acceleration_limits: true
    max_acceleration: 8.0
